WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -152,6 +152,16 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required] InputsRequ
 							return BadRequest($"Invalid ChangeOutputAddress Network.");
 						}
 					}
+					
+					if (round.GetAlicesBy(request.ChangeOutputAddress).ToArray().Length != 0)
+					{
+						return BadRequest($"Cannot register a ChangeOutputAddress twice.");
+					}
+
+					if (round.GetBobsBy(request.ChangeOutputAddress).ToArray().Length != 0)
```

Comment:
This won't help. See https://github.com/nopara73/WalletWasabi/blob/5f6bc59bfb59814e0eaf439209b36599fe54266d/WalletWasabi/Extensions/NBitcoinExtensions.cs#L188